packages/discord-bot/src/discord-listener/handlers.ts

```diff
@@ -60,7 +59,6 @@ export async function handleMessage(
     }
   } catch (error) {
     log.info(`Error Processing Discord Message`, error);
```

Comment:
NIT, can we turn this log level from info to error?

Reply:
Good catch, we need to continue reviewing how we log and process errors!